docker/conf/docker-compose-mac.yml.dist

```diff
@@ -24,7 +24,7 @@ services:
 
     php-fpm:
         build: project-base/docker/php-fpm
-        container_name: shopsys-framework-php-fpm
+        container_name: shopsys-framework
```

Comment:
Wow, again why? All containers have a name like this: prefix = shopsys_framework, postfix = name of the container. Now we have an exception for php-fpm. I do agree that name is too long, so why not name it something like ssfw-php-fpm or shopsys-php-fpm or just php-fpm (and also all others in the same way).

Reply:
It's the main container - every time you want to execute something in a container it's this container. Naming it just *shopsys-framework* should help people with this. Also it's quicker to use in the terminal: just write docker exec, press TAB and write your command. Comfy.

Reply:
So why do not use some of those suggestions I have made?

Reply:
The internal voting was 4:2 in favor of `shopsys-framework-php-fpm`. I've dropped the commit shortening the container name (7847cd651).